outputs.tf

```
@@ -44,6 +44,11 @@ output "lambda_function_source_code_size" {
   value       = try(aws_lambda_function.this[0].source_code_size, "")
 }
 
+output "lambda_function_url" {
```

```suggestion
output "lambda_function_arn" {
```

because this outputs `arn`.

Please add 2 more outputs for attributes of this resource - https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/lambda_function_url#attributes-reference

---

Still missing `lambda_function_arn` - https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/lambda_function_url#function_arn